pagopa/pn-frontend
# Feat/pn 7865

## Short description
Moved the pf login webapp under cittadini host

## List of changes proposed in this pull request
- Added homepage in the pf login package.json to build login dev under basepath auth
- Updated i18n configuration to load the localizations under the basepath auth
- Removed from pf login webapp those configurations that are useless
- Updated cdn configuration
    - added origin and cache behaviour in pf to link pfl webapp under path auth
    - added function to redirect login to cittadini webapp

## How to test
Go to cittadini and check that login is served under cittadini/auth.
In the browser, paste login url and check that the user is redirected to cittadini/auth
